gittrigger/gittrigger.go

```diff
@@ -35,12 +35,36 @@ func (g *gitTrigger) getLatestCommit() error {
 	return nil
 }
 
-func (g gitTrigger) checkTrigger() bool {
+func (g *gitTrigger) GetRemote() (string, error) {
```

Comment:
Because the remotes aren't part of the `gitTrigger` struct this function isn't really part of the GitTrigger package? Not really a hard objection here because this module is confusing to me.